torch/distributed/_tensor/ops/math_ops.py

```diff
@@ -251,3 +251,100 @@ def softmax_bwd_rule(op_schema: OpSchema) -> OutputSharding:
     ):
         raise RuntimeError("Cannot run _softmax_backward_data on sharding dimension!")
     return pointwise_rule(op_schema)
+
+
+def layer_norm_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> OpStrategy:
+    # args must be: input, normalized_shape, weight, bias, eps
```

Comment:
Good question. I did two things to testify:
1. `assert len(op_schema.args_schema) == 5` on line 258 to see if the schema is always a quintuple and the dtensor test has no error raised. 
2. However, this could be likely due to we always passed in `weight` and `bias` so I also added `assert bias is not None`. And the dtensor test raises error saying `bias` could be `None`. So we can tell that dtensor test actually tests the case where `bias` is optional.

Based on these two evidence, `weight`/`bias` always exist even if they're `None`.